lighthouse-core/gather/gatherers/js-usage.js

```diff
@@ -11,6 +11,22 @@ const Gatherer = require('./gatherer.js');
  * @fileoverview Tracks unused JavaScript
  */
 class JsUsage extends Gatherer {
+  constructor() {
+    super();
+    /** @type {LH.Crdp.Debugger.ScriptParsedEvent[]} */
+    this._scriptParsedEvents = [];
+    this.onScriptParsed = this.onScriptParsed.bind(this);
+  }
+
+  /**
+   * @param {LH.Crdp.Debugger.ScriptParsedEvent} event
+   */
+  onScriptParsed(event) {
+    if (event.sourceMapURL) {
```

Comment:
what does this have to do with anything?

Reply:
nada I copied from `source-maps`....